### DISCUSSION TITLE
q-file

### DISCUSSION THREAD

**Discussion by lukmagg** [quasarframework/quasar#12503] [General - Components / Directives / etc]
good morning people!

I am trying to send a pdf to codeigniter.

using q-file.

The problem is that when I get the pdf in the backend, it returns it to me as an object and I really don't know how to handle it.

does anyone have an idea.

Thanks in advance!!!

**Comment by MilosPaunovic** [comment]
Hello @lukmagg, that seems like the problem with the backend; not related to Quasar.

I'm guessing this could help you:
[https://codeigniter.com/user_guide/libraries/uploaded_files.html?highlight=files](https://codeigniter.com/user_guide/libraries/uploaded_files.html?highlight=files)

**Comment by lukmagg** [comment]
@MilosPaunovic 

My problem was that I didn't know exactly what format Quasar loads the files in q-file with.

But i already solved the problem.

I send the files with:

 const formData = new FormData();
      formData.set('file', this.pdf)
      
      axios({
        method: "POST",
        url: "my_path",
        data: form data,
        headers:{'Content-Type': 'multipart/form-data'}
      })


This has worked fine! Thank you!
